msurus/03_HTML/task_3_2.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="utf-8"> 
+		<title>Анкета</title>
+	</head>
+	<body>
+		<h1 align="center">Заполните анкету</h1>
+		<form id="anket" name="anketa" method="get" action="http://sc.it-academy.by/fe/form.php">
```

Comment:
not get, post

Reply:
sorry, forgot to change from the last task